### PR TITLE
Add support for OpenAI CLIP, and latest RADIO models

### PR DESCRIPTION
**Pull request by mranzinger**
The new RADIO models change the adaptor interface slightly to use a ModuleDict instead of a ModuleList. The consequence of this is that the state_dict changes slightly.

Also, in `examples`, it's now possible to load OpenAI CLIP models.

### CODE REVIEW

File: examples/zero_shot_imagenet.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def main(rank: int = 0, world_size: int = 1):
     parser.add_argument('--split', default='validation',
                         help='The dataset split to use.'
     )
-    parser.add_argument('--resize-multiple', type=int, default=None,
+    parser.add_argument('--resize-multiple', type=int, default=16,
```
Comment (gheinrich) [inline examples/zero_shot_imagenet.py:59]: I think `visualize_features` also needs this

Comment (gheinrich) [reply examples/zero_shot_imagenet.py:59]: `knn_classification` and `position_predictor` too, I think

Comment (mranzinger) [reply examples/zero_shot_imagenet.py:59]: Good catch. Done.

File: hubconf.py [hunk 2]
```diff
@@ -57,6 +60,10 @@ def radio_model(
     if key_warn.unexpected_keys:
         warnings.warn(f'Unexpected keys in state dict: {key_warn.unexpected_keys}')
 
+    if chk['args'].spectral_reparam:
+        disable_spectral_reparam(mod)
```
[the remark this answers is not in the document]
Comment (mranzinger) [reply hubconf.py:64]: It's a good question. I can see this being argued either way. We don't technically need to disable it, but it does increase memory usage and slow down the model a bit, given that the `weight` tensors are now a computed function. When disabling, it realizes the weight tensor, and then it's more like a regular ViT. Usually when I release a model, I manually disable spectral reparam for this reason.

What are your thoughts on this?

[the remark this answers is not in the document]
Comment (gheinrich) [reply hubconf.py:64]: That sounds reasonable indeed. Can you add a comment to make it more explicit?

**Review by gheinrich — APPROVED** [review]

File: hubconf.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mranzinger) [reply hubconf.py:71]: Makes sense. Added comment.

**Pull request merged**